### PR TITLE
explain how to run the project from git

### PR DESCRIPTION
**Pull request by dstathis**
fixes: #88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,16 @@ will produce a charm that looks like
...
 
 which should be all you need to `juju deploy` the charm!
+
+## How do I run it from source?
+
+    git clone https://github.com/canonical/charmcraft.git
+    virtualenv venv
+    . venv/bin/activate
+    pip install -r requirements.txt
+    PYTHONPATH=.:${PYTHONPATH} python charmcraft/main.py
```
Comment (facundobatista) [inline README.md:63]: `python -m charmcraft` is shorter and more to the point (no need to fix paths, or call internal modules)

File: README.md [hunk 2]
```diff
@@ -53,3 +53,16 @@ will produce a charm that looks like
...
+
+If you would like to run the tests you can do so with
+
+    pip install -r requirements-dev.txt
+    PYTHONPATH=.:${PYTHONPATH} ./run_tests
```
Comment (facundobatista) [inline README.md:68]: `./run_tests` should be enough, not path mangling needed...

File: README.md [hunk 3]
```diff
@@ -53,3 +53,16 @@ will produce a charm that looks like
             └── ...
 
 which should be all you need to `juju deploy` the charm!
+
+## How do I run it from source?
+
+    git clone https://github.com/canonical/charmcraft.git
```
Comment (chipaca) [inline README.md:59]:
```suggestion
    git clone https://github.com/canonical/charmcraft.git
    cd charmcraft
```

**Review by chipaca — CHANGES_REQUESTED** [review]
This is a good idea, but it needs a small tweak so you can actually copy-paste the code into a terminal (and people will).

File: README.md (hunk 3 above)
Comment (dstathis) [reply README.md:59]: Good catch

**Review by chipaca — APPROVED** [review]

**Pull request merged**